browser/layout/wrapper.html

```diff
@@ -33,10 +33,24 @@
 		<link rel="icon" type="image/png" href="https://www.ft.com/__assets/creatives/brand-ft/icons/v2/favicon-194x194.png" sizes="194x194">
 		<link rel="apple-touch-icon" sizes="180x180" href="https://www.ft.com/__assets/creatives/brand-ft/icons/v2/apple-touch-icon-180x180.png">
 
+		{{#if @root.flags.subscriberCohort}}
+		{{#unless @root.flags.disableIosSmartBanner}}
+		<meta name="apple-itunes-app" content="app-id=1200842933, affiliate-data=myAffiliateData, app-argument=myURL">
+		{{/unless}}
+		{{#unless @root.flags.disableAndroidSmartBanner}}
+		<link rel="manifest" href="/assets/manifest/manifest-v5-app-banner.json">
+		{{/unless}}
+		{{/if}}
+
 		{{#if @root.flags.disableAndroidSmartBanner}}
 		<link rel="manifest" href="/assets/manifest/manifest-v5.json">
-		{{else}}
-		<link rel="manifest" href="/assets/manifest/manifest-v5-app-banner.json">
+		<script>
```

Comment:
If this disables the banner, then could we not always load `/assets/manifest/manifest-v5-app-banner.json` and just put this script behind the flag?